### PR TITLE
Add French Language

### PR DESCRIPTION
**Pull request by Sheitak**
Hello,

Add French README,

Thank you,

### CODE REVIEW

**Comment by sabesansathananthan** [comment]
@esau-morais please review it

**Review by esau-morais — CHANGES_REQUESTED** [review]

**Review by esau-morais — CHANGES_REQUESTED** [review]
Thank you for contribute with French language!

_Note: You need to change the main `README.md`, removing the `Upcoming`_

**Comment by esau-morais** [comment]
If the French language not be changed until 1 week, @sabesansathananthan, you can close this PR.

_Note: One choice is review and make the changes, if @Sheitak not_

**Comment by sabesansathananthan** [comment]
@Sheitak Could you please consider the suggestions?

**Comment by esau-morais** [comment]
> @Sheitak Could you please consider the suggestions?

Probably he didn't see the review and is not so connected. I think it's necessary to close or solve by our own

**Comment by sabesansathananthan** [comment]
> > @Sheitak Could you please consider the suggestions?
> 
> Probably he didn't see the review and is not so connected to GitHub. I think it's good to close or solve by our own

Maybe he has some work so he couldn't open GitHub as well. shall we wait for a week? shall we close this on 14th Oct if @Sheitak didn't make changes?

**Comment by esau-morais** [comment]
> > > @Sheitak Could you please consider the suggestions?
> > 
> > 
> > Probably he didn't see the review and is not so connected to GitHub. I think it's good to close or solve by our own
> 
> Maybe he has some work so he couldn't open GitHub as well. shall we wait for a week? shall we close this on 14th Oct if @Sheitak didn't make changes?

Yes, sure

**Comment by Sheitak** [comment]
Hello gentlemen, excuse me I had an exam recently! I'm sorry for the delay, I'll take care of this right away @esau-morais @sabesansathananthan

File: translations/README.fr.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# GitHub Resume Generator
+
+![React GitHub Resume](./src/assets/readme/screenshot.png)
```
[the remark this answers is not in the document]
Comment (Sheitak) [reply translations/README.fr.md]: It's ok, I think

File: translations/README.fr.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+## Table of Content
+
+- [A propos](#about)
```
[the remark this answers is not in the document]
Comment (Sheitak) [reply translations/README.fr.md]: it's okay

File: translations/README.fr.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# GitHub Resume Generator
+
+![React GitHub Resume](./src/assets/readme/screenshot.png)
+
+## Table of Content
```
[the remark this answers is not in the document]
Comment (Sheitak) [reply translations/README.fr.md]: it's okay

File: translations/README.fr.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    yarn start
+    ```
+
+4. Visiter <http://localhost:3000>
+
+<h2 id='live-demo'><a href="https://react-github-resume.vercel.app/">Demo Direct</a></h2>
+
+[![Deployer avec Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/sabesansathananthan/React-GitHub-Resume)
+
+## 📄 License
```
Comment (sabesansathananthan) [inline translations/README.fr.md:71]: Is License is same as License in Fr??? For verification

**Review by sabesansathananthan — CHANGES_REQUESTED** [review]

**Comment by sabesansathananthan** [comment]
Please fill the PR form and Edit Main README and remove the word upcoming in French

**Comment by sabesansathananthan** [comment]
> Hello gentlemen, excuse me I had an exam recently! I'm sorry for the delay, I'll take care of this right away @esau-morais @sabesansathananthan

Good luck with your exams. <3

File: translations/README.fr.md (hunk 4 above)
Comment (Sheitak) [reply translations/README.fr.md:71]: Oh yes, forgotten, Licence*

File: translations/README.fr.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+Ce projet est sous licence MIT - voir le fichier [LICENSE](../LICENSE) pour plus de détails.
```
Comment (sabesansathananthan) [inline translations/README.fr.md:73]: Change [LICENSE] as well

**Review by sabesansathananthan — CHANGES_REQUESTED** [review]

File: translations/README.fr.md (hunk 5 above)
Comment (Sheitak) [reply translations/README.fr.md]: Noted

Comment (Sheitak) [reply translations/README.fr.md]: If other things are missing, do not hesitate, I put star

**Comment by sabesansathananthan** [comment]
Thank you for your PR... <3

**Pull request merged**